org.libretro.RetroArch.json

```diff
@@ -15,6 +15,7 @@
     "--device=all",
     "--filesystem=host",
```

Comment:
Related, do you think we should split up the `host` permission here? I'm open to any permissions to make it align with other flatpak standards.